GDJS/Runtime/events-tools/commontools.js

```diff
@@ -31,24 +30,15 @@ gdjs.evtTools.common.getVariableString = function(variable) {
 /**
  * @private
  */
-gdjs.evtTools.common.sceneVariableExists = function(
-  runtimeScene,
-  variableName
-) {
+gdjs.evtTools.common.sceneVariableExists = function(runtimeScene, variableName) {
   return runtimeScene.getVariables().has(variableName);
 };
 
 /**
  * @private
```

Comment:
Maybe try adding a description to the private fields explaining their purpose and why they are private could be useful (for devs).